src/taxonomy/import-tags/data/utils.js

```diff
@@ -0,0 +1,82 @@
+import messages from '../messages';
+import { importNewTaxonomy } from './api';
+
+// eslint-disable-next-line import/prefer-default-export
+export const importTaxonomy = async (intl) => {
+  /*
+    * This function is a temporary "Barebones" implementation of the import
+    * functionality with `prompt` and `alert`. It is intended to be replaced
+    * with a component that shows a `ModalDialog` in the future.
+    * See: https://github.com/openedx/modular-learning/issues/116
```

Comment:
^ Just flagging this in particular in case any reviewers are concerned about `alert()` and `prompt()` :) . This is just a temporary state during development to help get more PRs out faster and more iteratively, and will be replaced before any users ever see this (other than early beta testers).